test/CMakeLists.txt

```diff
@@ -1573,6 +1573,36 @@ add_custom_test(test_regression_opencl_float_mi100 VEGA_DISABLED HIP_DISABLED GF
     COMMAND	${ENVS_REGRESSION_ISSUE_1012} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --cmode conv --pmode default --group-count 1 --input 64,  64,  56, 56 --weights 256, 64,  1, 1 --pads_strides_dilations 0 0 1 1 1 1 ${ARGS_REGRESSION_ISSUE_1012}
 )
 
+
+set(CONV_CK_IGEMM_FWD_V4R4R4_XDLOPS_NHWC_ENV
+    MIOPEN_FIND_MODE=normal
+    MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvCkIgemmFwdV4r4r4XdlopsNhwc)
+
+add_custom_test(test_conv_ck_igemm_fwd_v4r4r4_xdlops_nhwc FLOAT_ENABLED HALF_ENABLED BF16_DISABLED  GFX908_ENABLED GFX90A_ENABLED SKIP_UNLESS_ALL
+COMMAND ${CONV_CK_IGEMM_FWD_V4R4R4_XDLOPS_NHWC_ENV}     $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --verbose --input 128 1024 14 14  --weights 2048 1024 1 1 --pads_strides_dilations 0 0 2 2 1 1 --disable-backward-data --disable-backward-weights  --in_layout NHWC --fil_layout NHWC --out_layout NHWC
```

Comment:
```
set(CONV_FORWARD_ONLY_NHWC
    --disable-backward-data --disable-backward-weights
    --in_layout NHWC --fil_layout NHWC --out_layout NHWC)
```
and use it in the new test cases instead of "--disable-backward-data --disable-backward-weights --in_layout NHWC --fil_layout NHWC --out_layout NHWC"

Reply:
done